app/src/main/res/layout/tool_chooser_menu.xml

```diff
@@ -8,157 +8,198 @@
+            android:layout_weight="1"
+            android:background="@android:color/transparent"
+            android:drawableStart="@drawable/ic_spray_can_icon"
+            android:paddingStart="15dp"
+            android:paddingEnd="15dp"
+            android:drawablePadding="15dp"
+            android:gravity="start|center_vertical"
+            android:onClick="chooseNewTool"
+            android:text="@string/drawSprayCan"
+            />
+
+    </LinearLayout>
+
+    <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_weight="1"
-        android:background="@android:color/transparent"
-        android:drawableStart="@drawable/ic_text"
-        android:paddingStart="15dp"
-        android:paddingEnd="15dp"
-        android:drawablePadding="15dp"
-        android:gravity="start|center_vertical"
-        android:onClick="chooseNewTool"
-        android:text="@string/drawTextButton" />
+        android:orientation="horizontal">
+
+        <Button
+            android:id="@+id/drawTextButton"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:background="@android:color/transparent"
+            android:drawableStart="@drawable/ic_text"
+            android:paddingStart="15dp"
+            android:paddingEnd="15dp"
+            android:drawablePadding="15dp"
+            android:gravity="start|center_vertical"
+            android:onClick="chooseNewTool"
+            android:text="@string/drawTextButton"
+            />
+
+        <Button
+            android:id="@+id/eraserButton"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:background="@android:color/transparent"
+            android:drawableStart="@drawable/ic_eraser_icon"
+            android:paddingStart="15dp"
+            android:paddingEnd="15dp"
+            android:drawablePadding="15dp"
+            android:gravity="start|center_vertical"
+            android:onClick="chooseNewTool"
+            android:text="@string/eraserButton"
+            />
 
-    <Button
-        android:id="@+id/cameraButton"
+    </LinearLayout>
+
+    <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_weight="1"
-        android:background="@android:color/transparent"
-        android:drawableStart="@drawable/ic_photo_camera_black_24dp"
-        android:paddingStart="15dp"
-        android:paddingEnd="15dp"
-        android:drawablePadding="15dp"
-        android:gravity="start|center_vertical"
-        android:onClick="chooseNewTool"
-        android:text="@string/drawCameraPicture" />
+        android:orientation="horizontal">
+
+        <Button
+            android:id="@+id/takePhoto"
```

Comment:
Please rename this id. The purpose of this button is to load an image from the gallery.